<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Poisson_Reconstruction_Dialog</class>
 <widget class="QWidget" name="Poisson_Reconstruction_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poisson Reconstruction Setup</string>
  </property>
  <widget class="QSpinBox" name="spinBox_threads">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_NormalSearchRadius">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_threads">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Threads</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NormalSearchRadius">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NormalSearchRadius</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_PoissonDepth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PoissonDepth</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_PoissonDepth">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
